docs/SUPPORTED_BROWSERS.md

```diff
@@ -0,0 +1,31 @@
+# Supported Browsers
+
+## Policy
+
+uPortal supports **the latest general audience release** and **one prior release** of these major browsers:
+
++ Chrome (on Android, MacOS, Windows) (including support for TalkBack)
++ Firefox (on MacOS and on Windows)
```

Comment:
Here's the thing: No browser support fairy comes along and makes it true that uPortal actually supports the browsers targeted in this policy.  Supporting browsers is work. I think the best strategy for uPortal is to commit to as little as we can get away with and then really fulfill those commitments. 

If it proves feasible, sustainable to fulfill a more modest browser support policy, then sure, uPortal could look to make bigger commitments. :)